network/src/connectivity_manager/mod.rs

```diff
@@ -118,6 +118,9 @@ pub enum ConnectivityRequest {
     UpdateEligibleNodes(DiscoverySource, HashMap<PeerId, HashSet<x25519::PublicKey>>),
     /// Gets current size of dial queue. This is useful in tests.
     #[serde(skip)]
+    GetConnectedSize(oneshot::Sender<usize>),
```

Comment:
man, i wish we could refactor these actors so we didn't need these test-only messages...

Reply:
I actually tried to refactor it, and then gave up to do this instead